src/shared/analytics/shared/addressActionToAnalytics.ts

```diff
@@ -99,17 +105,27 @@ export function addressActionToAnalytics({
     asset_amount_received: toMaybeArr(incoming?.map(convertQuantity)),
     asset_name_sent: toMaybeArr(outgoing?.map(getAssetName)),
     asset_name_received: toMaybeArr(incoming?.map(getAssetName)),
+    asset_address_sent: toMaybeArr(outgoing?.map(getAssetAddresses)),
+    asset_address_received: toMaybeArr(incoming?.map(getAssetAddresses)),
   };
   if (quote) {
     const zerion_fee_percentage = quote.protocol_fee;
     const feeAmount = quote.protocol_fee_amount;
     const asset = incoming?.[0]?.asset;
+    const output_chain = quote.output_chain;
+    const transaction_zerion_fee_asset_id = getFungibleAsset(asset)?.id;
```

Comment:
I think the name implies that this should be id of asset that the fee is paid with, but we assign the id of the incoming asset instead?

Reply:
In what case can incoming asset and zerion fee asset be different?

Reply:
The field is not required in the specs anymore (I validated with Anna) so I'll remove it rn, thank you!